config/environments/production.rb

```diff
@@ -63,7 +63,7 @@
 
   # Enable locale fallbacks for I18n (makes lookups for any locale fall back to
   # the I18n.default_locale when a translation can not be found)
-  config.i18n.fallbacks = true
+  config.i18n.fallbacks = [I18n.default_locale]
```

Comment:
Current day `I18n` installs warn you that this needs to change or things "may" break depending on how much this app relies on localizations.